website/pages/vmware/index.jsx

```diff
@@ -325,6 +325,29 @@ export default function VmwareIndex() {
           </ul>
         </div>
       </section>
+
+      <section id="faq" className={s.faq}>
+        <div className="g-grid-container">
+          <div className={s.tag}>Issues</div>
+          <h2 className={s.h2}>Known Issues</h2>
+          <ul>
+            <li>
+              <h4>Big Sur</h4>
+              <p className="g-type-body">
+                There are some notable issues for the VMWare provider on Big
+                Sur.
+                <ul>
+                  <li><a href="https://github.com/hashicorp/vagrant/issues/12045">Port forwarding</a></li>
+                  <li><a href="https://github.com/hashicorp/vagrant/issues/11682">Restoring snapshots</a></li>
+                  <li><a href="https://github.com/hashicorp/vagrant/issues/12052">Creating networking devices</a></li>
+                </ul>
+                Please also see the "Known Issues" section of the
+                <a href="https://docs.vmware.com/en/VMware-Fusion/12/rn/VMware-Fusion-12-Release-Notes.html"> VMWare 12 relese notes</a>.
```

Comment:
```suggestion
                 <a href="https://docs.vmware.com/en/VMware-Fusion/12/rn/VMware-Fusion-12-Release-Notes.html">VMWare 12 relese notes</a>.
```

Reply:
This change makes the spacing a little funky

<img width="158" alt="Screen Shot 2021-01-12 at 11 56 26 AM" src="https://user-images.githubusercontent.com/976973/104353180-3e965300-54cd-11eb-89f8-75b3cb2ba608.png">

Reply:
Ah yeah was worried about that, please ignore!